src/component/calculator.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+
+export default class Calculator extends Component {
```

Comment:
- [ ] OPTIONAL: For React component files, it is customary to capitalize them like this: `Calculator.js`.

Reply:
Thank you for this information @Meltrust

Reply:
@Meltrust  capitalized Calculator.js
![e51](https://user-images.githubusercontent.com/40554384/164270626-d11789e6-896f-4e41-b85c-0e0a25a12812.PNG)